apps/src/applab/applab.js

```diff
@@ -1445,7 +1413,7 @@ Applab.onPuzzleComplete = function (submit) {
         level: level.id,
         result: levelComplete,
         testResult: Applab.testResults,
-        submitted: !!submit,
+        submitted: submit ? "true" : false,
```

Comment:
So it should be fine if GameLab and AppLab just return a boolean.  When sending the report the boolean will be turned into the appropriate query string to be posted to the server.  But we will then see this warning -- https://github.com/code-dot-org/code-dot-org/commit/b7722bbaa31ec562fd5ac31b57529c12ca56aa8f#diff-2efe6c81fa55b55b1c3112a31b198e31R146 -- unless you think the warning should be relaxed?  Not sure if it should still be in place for other game types...